src/server/main.py

```diff
@@ -19,9 +20,12 @@ def main():
     args = parser.parse_args()
 
     try:
+        now = datetime.now()
+        time_stamp = now.strftime("%m.%d.%Y_%H-%M-%S")
+        file_name = args.f + f'output_{time_stamp}.json'
```

Comment:
For now we aren't creating a file in main. This will be another ticket, so remove this line.

Reply:
Which line?

Reply:
`file_name = args.f + f'output_{time_stamp}.json'`
This one.

Reply:
If we aren't creating a file in main, I'll remove the write completely. The consume function still needs the filename for its own use.